src/SpinWaveTheory/SpinWaveTheory.jl

```diff
@@ -205,6 +209,12 @@ function swt_data_dipole!(sys::System{0})
         # Precalculate operators for rotating Stevens coefficients.
         V = operator_for_stevens_rotation(2, R)
 
+        # Observables are 1x3 row vectors
+        for μ = 1:num_observables(obs)
+            row = observable_at_site(obs.observables[μ],CartesianIndex(1,1,1,atom))
+            observables_localized[:,:,μ,atom] .= Matrix(row) * R
```

Comment:
If `row` is a row vector, how is Matrix(row) * R a 2-dimensional object? (I think I get the big idea, but I'm a bit confused by the implementation.)

Reply:
It's a `1x3` matrix, so it's teeeeeechnically 2-dimensional. The `size(observables_localized,1)` is `1`

Reply:
Gotcha. Thanks.